src/ORM/Association/BelongsToMany.php

```diff
@@ -218,7 +216,7 @@ public function getForeignKey()
      * @param mixed $sort A find() compatible order clause
      * @return $this
      */
-    public function setSort($sort)
+    public function setSort(mixed $sort)
```

Comment:
This should probably match Query::order()?

Reply:
Updated setSort() and getSort(). getSort() allows returning null.